exporter/elasticsearchexporter/integrationtests/Makefile

```
@@ -0,0 +1,4 @@
+.PHONY: test
+test:
+	go clean --testcache
+	go test -v -timeout=10m ./...
```

Are we sure we need this file? I was able to remove it (and the `go.*` files in this folder), run `make test` from the `exporter/elasticsearchexporter` folder and get the integration tests to run.

---

BTW, I don't have a strong preference here.  I understand the benefits of isolating the integration testing code and dependencies from the exporter's code, unit tests, and dependencies.  Looking through other components, I found a similar pattern in use by the [`datadogexporter`'s integration tests](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/datadogexporter/integrationtest) so I don't think there's too much of a precedent in either direction.  I'm happy to keep things the way you have them but maybe, just for consistency, we could make two small changes:
* Rename this folder from `integrationtests` -> `integrationtest`, and
* See if we can get away with the `Makefile` in this folder to simply include the top-level `Makefile.Common` file so:
   ```suggestion
   include ../../../Makefile.Common
   ```

---

> Are we sure we need this file?

Good point. I think we don't need this file but we should keep the integration tests as a separate module (reasoning explained below).

> and the go.* files in this folder

I assume you are referring to the `go.mod` and `go.sum` files. I created `integrationtests` as a separate module to keep the extra dependencies used in the tests contained and isolated -- this could potentially avoid any dependency conflicts with the original module and also give more freedom to use dependencies for the tests while writing the tests.

> run make test from the exporter/elasticsearchexporter folder and get the integration tests to run.

Keeping the above point in context, I think we can remove the `Makefile` from the integration tests module and update the exporter's Makefile to execute the test while keeping the integration tests in a separate module. Does this make sense to you?

---

Ah github didn't load your latest comment for me 😅 

> Rename this folder from integrationtests -> integrationtest, and
> See if we can get away with the Makefile in this folder to simply include the top-level Makefile.Common file so:

ACK on both points, for the second point I will see if we can drop the Makefile in the integration test entirely.

---

Yup, that sounds good to me. For consistency, we could also take the approach used by the `datadogexporter` (see my previous comment in this thread).

---

Thanks for the tip, I kept it same as `datadogexporter` (and others) by just importing the common Makefile.